ext/tags.go

```diff
@@ -55,6 +65,9 @@ var (
 	// PeerService records the service name of the peer
 	PeerService = stringTagName("peer.service")
 
+	// PeerAddress records the address name of the peer
```

Comment:
this should have a more detailed comment since it's easy to misinterpret... maybe just adapt the text from the spec?